extensions/sql-database-projects/src/common/utils.ts

```diff
@@ -45,3 +46,27 @@ export function trimChars(input: string, chars: string): string {
 
 	return output;
 }
+
+
+/**
+ * Trims file name from @param input and return UriPath
+ */
+export function trimFileName(input: vscode.Uri): vscode.Uri {
+	let lastLevel: number = input.path.lastIndexOf('/');
+	if (lastLevel < 1) {
+		return input;
+	}
+	return vscode.Uri.file(input.path.substring(0, lastLevel));
```

Comment:
can probably use path.dirname() instead of finding the index of the last '/'  and the substring

Reply:
Done.